src/platform/baytrail/include/platform/drivers/dw-dma.h

```diff
@@ -48,6 +48,10 @@
 #define platform_dw_dma_set_transfer_size(chan, lli, size) \
 	(lli->ctrl_hi |= (size & DW_CTLH_BLOCK_TS_MASK))
 
+#define platform_dw_dma_llp_config(dma, chan, config)
```

Comment:
Could we use the UNUSED macros here, similar to how they were added for the trace_* macros?